src/editors/_shared/hooks/DataTypes/types/StringType.js

```diff
@@ -23,20 +22,8 @@ class StringType extends DataType {
    * @param {*} value
    * @returns
    */
-  validate(value) {
-    return new Promise(resolve => {
-      try {
-        if (checkIfDefaultOrDisabled(value)) {
-          return resolve({ success: true, value });
-        }
-        const parsed = this.getParsedValue(value);
-        const isValid =
-          typeof parsed === DATA_TYPES.STRING || parsed instanceof String;
-        resolve({ success: isValid });
-      } catch (e) {
-        resolve({ success: false, error: "MandatoryStringQuotes" });
-      }
-    });
+  validate(_value) {
+    return Promise.resolve({ success: true });
```

Comment:
I don't like that we aren't validating this anymore. there were a lot of validations taking place. Removing these validations might end up causing regressions. Can you please confirm why this was there in the first place and if it is safe to remove?

Reply:
It's really doing nothing. Since strings are always strings now.